tensorflow/lite/micro/kernels/unidirectional_sequence_lstm_test.cc

```diff
@@ -34,6 +35,1854 @@ constexpr int max_num_input_tensors = 24;
 constexpr int output_tensor_index = max_num_input_tensors;
 constexpr int intermediate_tensor_base = max_num_input_tensors + 1;
 
+// data for no CIFG, no peephole, no projection, and no clipping
+
+constexpr int n_batch_no_cifg_no_peephole_no_proj_no_clipping = 1;
+constexpr int n_input_no_cifg_no_peephole_no_proj_no_clipping = 2;
+// n_cell and n_output have the same size when there is no projection.
+constexpr int n_cell_no_cifg_no_peephole_no_proj_no_clipping = 4;
+constexpr int n_output_no_cifg_no_peephole_no_proj_no_clipping = 4;
+constexpr int sequence_length_no_cifg_no_peephole_no_proj_no_clipping = 3;
+
```

Comment:
If we define a TestConfig (see later comments), then we don't need to define all these individual constants that are long. We can just TestConfig NoCifgNoPeeholeNoProjNoClippingTestConfig.xxx = xxxx? WDYT?

Reply:
Same as above.